versioned_docs/version-8.7/apis-tools/camunda-api-rest/camunda-api-rest-authentication.md

```diff
@@ -121,7 +121,7 @@ curl --header "Authorization: Bearer ${TOKEN}" \
      ${ZEEBE_REST_ADDRESS}/v2/topology
 ```
 
-A successful response includes [information about the cluster](/apis-tools/camunda-api-rest/specifications/get-topology.api.mdx). For example:
+A successful response includes [information about the cluster](/apis-tools/camunda-api-rest/specifications/get-cluster-topology.api.mdx). For example:
```

Comment:
The PR includes a small handful of changes like this -- 8.7 docs that are referencing endpoints that are named differently in the stable/8.7 spec.

Reply:
Is the basis for our 8.7 docs the 8.6 docs or the 8.8 docs? I assume we branched off the former "next" into "next 8.7"? That would explain something like this.

There might be other things that "suffer" from this then, I could imagine. The former "next" docs contained a lot of stuff already that are not really meant for the "new" 8.7 now.